src/Assign/README.md

```diff
@@ -0,0 +1,87 @@
+# Assign
+```haskell
+Assign Object
+```
+`Assign` is a `Monoid` that will combine two objects using `Object.assign` on their `Object` values, merging and overwriting new values for values that are not undefined.
+
+```js
+const Assign = require('crocks/Assign')
+
+const first = { name : "Bob" }
+const last = { lastname: "Smith" }
+
+const person = Assign(first).concat(last) //=> Assign { name: "Bob", lastName: "Smith" }
+
+```
+## Implements
+`Semigroup`, `Monoid`
+
+## Constructor Methods
+
+### empty
+```haskell
+Assign.empty :: () -> Assign
+```
+
+`empty` provides the identity for the `Monoid` in that when the value it provides is `concat`ed to object other value, it will return the other value. In the case of `Assign` the result of `empty` is `{}`. `empty` is available on both the Constructor and the Instance for convenience.
+```js
+Assign.empty() //=> Assign true
+
+Assign({}).concat(Assign.empty())   //=> Assign {}
+Assign({ a: 1 }).concat(Assign.empty())  //=> Assign { a: 1 }
+```
+
+
+### type
+```haskell
+Assign.type :: () -> String
+```
+
+`type` provides a string representation of the type name for a given type in `crocks`. While it is used mostly internally for law validation, it can be useful to the end user for debugging and building out custom types based on the standard `crocks` types. While type comparisons can easily be done manually by calling `type` on a given type, using the `isSameType` function hides much of the boilerplate. `type` is available on both the Constructor and the Instance for convenience.
+
+```js
+const Assign = require('crocks/Assign')
+const isSameType = require('crocks/predicates/isSameType')
+
+Assign.type() //=>  "Assign"
+
+isSameType(Assign, Assign({}))         //=> true
+isSameType(Assign, Assign)            //=> true
```

Comment:
I tried to keep it consistent with the other docs here.  It might be useful to play around with the different use cases early on, but I don't know what the right answer is.

Reply:
Oh snap, good catch. That is a problem, go ahead and remove it from your additions. I will take care of the existing ones